kotlin-android/app/src/main/java/com/mongodb/tasktracker/LoginActivity.kt

```diff
@@ -47,14 +51,17 @@ class LoginActivity : AppCompatActivity() {
 
 
         if (createUser) {
+            // register a user using the Realm App we created in the TaskTracker class
             taskApp.emailPasswordAuth.registerUserAsync(username, password) {
+                // while this operation completes, disable the buttons to login or create a new account
                 binding.buttonCreate.isEnabled = true
```

Comment:
Set `isEnabled = true` to disable?
```suggestion
                binding.buttonCreate.isEnabled = false
```